Formula/djl-serving.rb

```diff
@@ -0,0 +1,48 @@
+class DjlServing < Formula
+  desc "This module contains an universal model serving implementation"
+  homepage "https://github.com/awslabs/djl/tree/master/serving"
+  url "https://djl-ai.s3.amazonaws.com/publish/djl-serving/serving-0.11.0.tar"
+  sha256 "cd57bf4cf4552ad749338ef76074d4819a0ff45e849321b4ffaf38ebb03c86e9"
+  license "Apache-2.0"
+
+  bottle :unneeded
+
+  depends_on "openjdk"
+
+  def install
+    # Install files
+    rm_rf Dir["bin/*.bat"]
+    mv "bin/serving", "bin/djl-serving"
+    libexec.install Dir["*"]
+    (bin/"djl-serving").write_env_script "#{libexec}/bin/djl-serving", JAVA_HOME: Formula["openjdk"].opt_prefix
+  end
+
+  plist_options manual: "djl-serving"
+
+  def plist
+    <<~EOS
+      <?xml version="1.0" encoding="UTF-8"?>
+      <!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+      <plist version="1.0">
+        <dict>
+          <key>Disabled</key>
+          <false/>
+          <key>Label</key>
+          <string>#{plist_name}</string>
+          <key>ProgramArguments</key>
+          <array>
+            <string>#{bin}/djl-serving</string>
+            <string>run</string>
+          </array>
+          <key>KeepAlive</key>
+          <true/>
+        </dict>
+      </plist>
+    EOS
+  end
+
+  test do
+    assert_predicate testpath/"#{bin}/djl-serving", :exist?
+    assert_predicate testpath/"#{libexec}/lib/serving-0.11.0.jar", :exist?
```

Comment:
Can you propose a more substantial test than checking for the existence of some files? These tests rarely ever fail even for broken installations.

>We want tests that don’t require any user input and test the basic functionality of the application.

Ref. https://docs.brew.sh/Formula-Cookbook#add-a-test-to-the-formula